### PR TITLE
Backend for adding image complete

### PR DESCRIPTION
**Pull request by wilsonwtcheng**

### CODE REVIEW

File: app/assets/javascripts/angular/controllers/posts/index.js.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+app.controller('PostCtrl',['$scope', '$http', '$location', function($scope, $http, $location) {
+
```
Comment (harrychenca) [inline app/assets/javascripts/angular/controllers/posts/index.js.erb]: Indentations

File: app/assets/javascripts/angular/controllers/posts/index.js.erb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  $scope.form = {};
+  // $scope.user = {};
+  var url = "<%= ENV['URL'] %>" || "http://localhost:3000/";
+  
+    $http.get(url + 'posts').success(function(response){
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/posts/index.js.erb]: Actual indentation error

File: app/assets/javascripts/angular/controllers/posts/index.js.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+      console.log(response)
+
+      $scope.posts = response;
+
+      $scope.form.create = function(index){
+        console.log($scope.posts);
+        $scope.posts[index]
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/posts/index.js.erb]: What is this for?

File: app/assets/javascripts/angular/controllers/posts/new.js.erb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+	$scope.form = {};
+	$scope.post = {};
+	$scope.picture= {};
+
+	var url = "<%= ENV['URL'] %>" || "http://localhost:3000/";
+	
+	//this .create matches with the .create in form.create in news.html
+	$scope.form.create = function() {
+		var data = {
```
Comment (harrychenca) [inline app/assets/javascripts/angular/controllers/posts/new.js.erb]: Indentation

File: app/assets/javascripts/angular/controllers/posts/new.js.erb [hunk 5]
```diff
@@ -0,0 +1,34 @@
+app.controller('PostNewCtrl',['$scope', '$http', '$location', 'Upload', function($scope, $http, $location, Upload) {
+
```
Comment (harrychenca) [inline app/assets/javascripts/angular/controllers/posts/new.js.erb]: Indentations

File: app/assets/templates/posts/userpost.html [hunk 6]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (harrychenca) [inline app/assets/templates/posts/userpost.html]: Remove line breaks when we don't need them

**Comment by h8rry** [comment]
Please gitignore `/public/system`

File: app/assets/javascripts/angular/controllers/posts/index.js.erb [hunk 7]
```diff
@@ -0,0 +1,30 @@
+app.controller('PostCtrl',['$scope', '$http', '$location', function($scope, $http, $location) {
+
+  $scope.form = {};
+  // $scope.user = {};
```
Comment (f3r) [inline app/assets/javascripts/angular/controllers/posts/index.js.erb]: remove commented line

File: app/assets/javascripts/angular/controllers/posts/userpost.js.erb [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    $http.get(url + 'posts').success(function(response){
+      console.log(response)
+
+      $scope.posts = response;
+
+      $scope.form.create = function(index){
+        console.log($scope.posts);
+        $scope.posts[index]
+
+      //  var data = {
```
Comment (h8rry) [inline app/assets/javascripts/angular/controllers/posts/userpost.js.erb]: Please remove these comments

File: app/assets/templates/posts/index.html [hunk 9]
```diff
@@ -0,0 +1,62 @@
...
+      <div class="page-header">
+      Title:  {{ post.title }}
+      Content:  {{ post.content }}
+      <!-- Category  {{ post.category }} -->
+        <small>
+          {{ post.created_at | date }}
+        </small>
+      </div>
+
+<!-- 		  <ul class="list-group">
```
Comment (h8rry) [inline app/assets/templates/posts/index.html]: Please remove these comments

**Comment by h8rry** [comment]
Looks good :+1:

**Pull request merged**